Wikipedia/Code/WMFChangePasswordViewController.swift

```diff
@@ -101,23 +101,20 @@ class WMFChangePasswordViewController: WMFScrollViewController, Themeable {
         enableProgressiveButton(false)
         WMFAlertManager.sharedInstance.dismissAlert()
                 
-        guard let userName = userName else {
+        guard let userName = userName,
+            let password = passwordField.text,
+            let retypePassword = retypeField.text else {
```

Comment:
i tested changing a pwd and still works fine, but may want assert here just in case